#Define goal's message
string turtle_name
float32 speed
float32 distance
bool isForward
---
#Define result's message
---
#Define feedback's message
turtlesim/Pose pose
